database/rrdcalc.h

```diff
@@ -223,6 +223,9 @@ static inline int rrdcalc_isrepeating(RRDCALC *rc) {
     return 0;
 }
 
-#define MAX_RRDCALC_VARIABLE 100
+#define RRDCALC_VAR_MAX 100
+#define RRDCALC_VAR_FAMILY "$family"
+#define RRDCALC_VAR_LABEL "$label:"
+#define RRDCALC_VAR_LABEL_LEN 7
```

Comment:
Considering that it is possible the previous variable to be changed in future PRs, I suggest to rewrite this last to:

```c
#define RRDCALC_VAR_LABEL_LEN (sizeof(RRDCALC_VAR_LABEL)-1)
```

Reply:
Thanks Thiago, changed!